rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,257 @@
+
+    - name: resources
+      level: extended
+      type: nested
+      description: >
+        If the PE contains resources, some info about them
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: -1
+
+        - name: filetype
+          level: extended
+          description: File type of the resources section.
+          type: keyword
+          example: Data
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the resources section.
+          type: long
+          example: 0, 1
+
+        - name: sha256
+          level: extended
+          description: SHA256 hash of resources section.
+          type: keyword
+          example: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+        - name: language
+          level: extended
+          description: Language identification.
+          type: keyword
+          example: "CHINESE SIMPLIFIED"
+
+        - name: types
+          level: extended
+          type: keyword
+          short: List of resource types.
+          description: >
+            Digest of resource types. Key is resource type (as string) and value
+            is how many resources there are of that specific type (as integer)
+          example: { "RT_VERSION": 1, "RT_MANIFEST": 1 }
+
+    - name: exports
+      level: extended
+      type: keyword
+      description: >
+        List of symbols exported by PE
+      example: DllInstall, DllRegisterServer, DllUnregisterServer
+
+    - name: creation_date
+      level: extended
+      short: Build or compile date.
+      description: >
+        Extracted when possible from the file's metadata. Indicates when it was
+        built or compiled. It can also be faked by malware creators.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: authentihash
+      level: extended
+      description: >
+        Authentihash of the PE file.
+      type: keyword
+      example: ac9555d914bbb112ecc5f15bb9887ca8371f493ab0941344e976bb8410c8aa78
+
+    - name: compile_timestamp
+      level: extended
+      description: >
+        Compile timestamp of the PE file.
+      type: date
+      example: "2020-11-05T17:25:47.000Z"
+
+    - name: compiler
+      level: extended
+      description: >
+        Compiler information for the PE file.
+      type: object
+      fields:
+        - name: name
+          level: extended
+          type: keyword
+          description: >
+            Name of the compiler
+          example: Clang
+
+        - name: version
+          level: extended
+          type: keyword
+          description: >
+            Version of the compiler.
+          example: 11.0.0
+
+    - name: rich_header
+      level: extended
+      description: >
+        Rich Header information for the PE file.
+      type: object
+      fields:
+        - name: hash
```

Comment:
Does it make sense to reuse the existing `hash` fieldset here?

Reply:
Agreed, and I think that was the point of adjusting this to be an object too. Any chance you know how to document that in the RFC doc @rw-access ?

Reply:
Good callout @rw-access and @andrewstucki 

I should also remove it here and use the existing `hash` fieldset, right?

https://github.com/elastic/ecs/blob/4fa79d5321ba6674d652c292131f2bf5f47a48f9/rfcs/text/pe/pe.yml#L11-L43

Reply:
If the intent is still to reuse `hash.*`, let's remove the now unneeded hash fields and note the intent to nest `hash.*` fields in the doc. We can sort out the declaration in the field definitions in the next stage's PR.

Reply:
Thanks, fix't.